tests/repo/x86_64/setup.ini

```diff
@@ -0,0 +1,20 @@
+# This file was automatically generated at 2023-04-06 23:08:13 CEST.
+#
+# If you edit it, your edits will be discarded next time the file is
+# generated.
+#
+# See https://sourceware.org/cygwin-apps/setup.ini.html for a description
+# of the format.
+release: cygwin
```

Comment:
I did not know this, thank you. I will consider it for my neovim overlay package server.

As for the creation process: I wrote the shell script and tar'ed it up, than wrote the .hint file manually and ran `mksetupini --arch x86_64 --inifile=x86_64/setup.ini --releasearea=. --disable-check=missing-required-package,missing-depended-package` to generate the `setup.ini`.

Reply:
oh and that complained about a missing source package so I tar'ed the script again and wrote a second .hint file...

Reply:
> and ran mksetupini [...] to generate the setup.ini.

Thanks.  This seems like a stupidity in mksetupini, defaulting to release:cygwin rather than omitting it, so I'll fix that.

Reply:
You may also want to update the "Creating an overlay Cygwin package server" section in https://cygwin.com/package-server.html#overlay to specify a custom release string.